test_scripts/unit_tests/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM ubuntu:18.04
+SHELL ["/bin/bash", "-c"]
+RUN apt-get update
+RUN apt-get install -y python procps git
+RUN git clone https://github.com/bats-core/bats-core.git /bats/
+RUN /bats/install.sh /usr/local
```

Comment:
Generally it's good to combine the RUN statements for getting and installing, then removing.
It reduces the layers and sizes of the layers.

Reply:
Good to know!